packages/roosterjs-content-model/test/modelApi/selection/adjustWordSelectionTest.ts

```diff
@@ -0,0 +1,280 @@
+import { adjustWordSelection } from '../../../lib/modelApi/selection/adjustWordSelection';
+import { ContentModelDocument } from '../../../lib/publicTypes/group/ContentModelDocument';
+import { ContentModelSegment } from '../../../lib/publicTypes/segment/ContentModelSegment';
+
+const defaultMarker: ContentModelSegment = {
+    segmentType: 'SelectionMarker',
+    format: {},
+    isSelected: true,
+};
+
+describe('adjustWordSelection', () => {
+    function runTest(
+        model: ContentModelDocument,
+        result: ContentModelSegment[],
```

Comment:
Suggest we also verify the model object since in your code you will split some segments.

Reply:
Which content model object?

Reply:
function runTest(
  model,  <================ model will be changed during the test, so we should also verify the changing result is expected.
  result,
)

Reply:
That feels like a test for splitTextSegment not for adjustWordSelection.

Reply:
Yes, you can say that. But is there a dedicate test for splitTextSegment? If not, you can also cover it here, given that splitTextSegment() is not a public function.